packages/react-examples/.storybook/preview.js

```diff
@@ -149,6 +186,11 @@ function generateStoriesFromExamples(key, stories, req) {
   const story = stories.get(componentName);
   const exampleModule = /** @type {(key: string) => ComponentModule} */ (req)(key);
 
+  if (!story) {
```

Comment:
properly guarding/narrowing union -> without this we had  `undefined` in the union